keyboards/kelwin46.5/keymaps/default/keymap.c

```diff
@@ -0,0 +1,170 @@
+#include "kb.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```

Comment:
These need to be `LAYOUT` and everything but the first two layers needs to be removed.